login-form/src/components/page-components/Login.js

```diff
@@ -0,0 +1,184 @@
+            <RebassLabel
+              fontSize={[2, 3, 4]}
+              marginBottom={[2, 3, 4]}
+              name={"Email"}
+              htmlFor="email"
+            />
+            <Input
+              sx={{
+                fontFamily: "monospace",
+                border: "none",
+                borderWidth: "1px",
+                borderBottom: "solid",
+                borderColor: "black",
+                ":focus": {
+                  outline: "none",
+                },
+              }}
+              fontSize={[2, 3, 4]}
+              mb={[2, 3, 4]}
+              value={userName}
+              onChange={(e) => setUserName(e.target.value)}
+              id="email"
+              name="email"
+              type="email"
+              placeholder="Email address"
+            />
+          </Box>
+          <Box>
+            <RebassLabel
+              fontSize={[2, 3, 4]}
+              marginBottom={[2, 3, 4]}
+              name={"Password"}
+              htmlFor="password"
+            />
+            <Input
+              sx={{
+                fontFamily: "monospace",
+                border: "none",
+                borderWidth: "1px",
+                borderBottom: "solid",
+                borderColor: "black",
+                ":focus": {
+                  outline: "none",
+                },
+              }}
+              mb={[2, 3, 4]}
+              fontSize={[2, 3, 4]}
+              value={passWord}
+              onChange={(e) => setPassWord(e.target.value)}
+              id="password"
+              name="password"
+              type="password"
+              placeholder="Password"
+            />
+          </Box>
+          <Box>
+            <Button
+              mt={[2, 3, 4]}
+              mb={[2, 3, 4]}
+              pt={[3]}
+              pr={[3, 4, 5]}
+              pl={[3, 4, 5]}
+              pb={[3]}
+              type="submit"
+              component={Link}
+              to="/dashboard"
+              bg="#495464"
+              onClick={(e) => loginClicked(e)}
+            >
+              Login
+            </Button>
+
+            <br />
+            <Box color="red">
+              {/* In case of wrong credentials */}
+              {responseMessage}
+            </Box>
+          </Box>
+          {/* <div>
+            {props.grandTotal}
+          </div> */}
+        </form>
+      </Box>
+    </Box>
+  );
+};
+
+const mapStateToProps = (state) => {
+  console.log("state", state);
+  return {
+    grandTotal: state.example.total,
+    userName: state.login.userName,
+    passWord: state.login.passWord,
+    loginResponse: state.sagaExample?.loginResponse?.status,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    incrementBy10: () => {
+      dispatch(incrementBy(10));
+      console.log("Test");
+    },
+
+    loginNow: (userName, passWord) => {
+      dispatch(login(userName, passWord));
+    },
+    loginSaga: (userName, passWord) => {
```

Comment:
proper naming for variables is important, this will improve the readability. Within the component, there is no meaning for saga.

```suggestion
    loginAction: (userName, passWord) => {
```
```suggestion
    submitLoginDetails: (userName, passWord) => {
```